packages/wallet-ui/src/components/ui/organism/AccountSwitchModal/HiddenAccountsList/HiddenAccountsList.view.tsx

```diff
@@ -0,0 +1,51 @@
+import { Account } from 'types';
+import {
+  AccountImageStyled,
+  AccountSwitchMenuItem,
+  MenuSection,
+  MenuItemText,
+} from '../AccountSwitchModal.style';
+import { Menu } from '@headlessui/react';
+import { Box, IconButton } from '@mui/material';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import { shortenAddress } from 'utils/utils';
+
+interface Props {
+  accounts: Account[];
+  unHideAccount: (params: { chainId: string; address: string }) => void;
+  setShowHiddenAccounts: (value: boolean) => void;
+  chainId: string;
+}
+
+export const HiddenAccountsListView = ({
+  accounts,
+  unHideAccount,
+  setShowHiddenAccounts,
+  chainId,
+}: Props) => (
+  <MenuSection style={{ height: 201, overflowY: 'auto' }}>
+    {accounts.map((account) => (
+      <Menu.Item key={account.address}>
+        <AccountSwitchMenuItem
+          style={{ paddingLeft: 22, opacity: 0.5 }}
+          onClick={(e) => {
+            setShowHiddenAccounts(false);
+            unHideAccount({ chainId, address: account.address });
+            e.preventDefault();
```

Comment:
you can lets this view focus on render, for event click, you can have a function come from parent
and e.preventDefault(); go first usually
wdyt?
```suggestion
            e.preventDefault();
            onAccountVisibleClick(account);
```